projects/demo/src/modules/app/home/examples/angular-json-styles.md

```diff
@@ -5,12 +5,7 @@
       "architect": {
         "build": {
           "options": {
-            "styles": [
-              "node_modules/@taiga-ui/core/styles/taiga-ui-global.less",
-              "node_modules/@taiga-ui/core/styles/taiga-ui-theme.less",
-              "node_modules/@taiga-ui/core/styles/taiga-ui-fonts.less",
-              "src/styles.css"
-            ]
+            "styles": ["node_modules/@taiga-ui/core/styles/taiga-ui-theme.less", "src/styles.css"]
```

Comment:
Missing fonts, once you return them to core styles.

Reply:
fixed